spec/requests/topics_controller_spec.rb

```diff
@@ -1792,6 +1792,22 @@ def fabricate_topic(user, category = nil)
     end
   end
 
+  describe '#show_by_external_id' do
+    before_all do
+      Fabricate(:topic, external_id: 'asdf')
+    end
```

Comment:
Is there a reason we use a `before_all` block here instead of `fab!` which does the same thing?

Reply:
Hey, me and @oblakeerickson paired on this. I don't have a strong preference, but `fab!` creates a name that wouldn't be used in this case, so it would just be noise.